src/main/java/HttpProtocal.java

```diff
@@ -0,0 +1,8 @@
+package com.td.HttpServer;
+
+public interface HttpProtocal {
```

Comment:
Okay, I understand the bad naming convention.  What I'm confused on is when testing a class that is injected with a HttpRequest. The examples I've seen for making your own mocks have the class take an interface and the mock just then implements that interface. Should I just have the mock class I create in the test be named the same as the class that would be injected??